ParProcCo/job_scheduler.py

```diff
@@ -129,11 +95,13 @@ class StatusInfo:
 
 
 class JobScheduler:
+    RE_CPU = re.compile(r"cpu=(\d+)")
+    RE_GPU = re.compile(r"cpu=(\d+)")
+
```

Comment:
I think this is a typo
```suggestion
    RE_GPU = re.compile(r"gpu=(\d+)")
```